scripts/modules/cli.py

```diff
@@ -677,6 +684,12 @@ def start_handler(self):
                 up_params.extend(["--quiet-pull"])
             self.run_docker_compose_process(docker_compose_cmd + up_params)
 
+        if args.get("enable_elastic_agent", False) and args["elastic-agent-apm-binary-path"]:
+            is_snapshot = args["elastic_agent_snapshot"] or not (any((args["elastic_agent_bc"], args["elastic_agent_release"])))
```

Comment:
thePython lintin failed `[2021-01-18T14:14:04.059Z] scripts/modules/cli.py:688:121: E501 line too long (128 > 120 characters)`

```suggestion
            is_snapshot = args["elastic_agent_snapshot"] or not (
                any((args["elastic_agent_bc"], args["elastic_agent_release"])))
```